src/pkcs11_1000.c

```diff
@@ -19,7 +19,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <utee_defines.h>
```

Comment:
Please take into account my commit: https://github.com/xen-troops/zephyr-optee-test/pull/5/commits/7858566823b417267ed5e058f8b77c7609279bd7
Probably it's better to remove util.h

Reply:
Actually, utilities from `util.h` are used in tests. At least `ARRAY_SIZE`.